bool can_shoot
int16[] line_of_sight_start
int16[] line_of_sight_end

int16[] distances
int16 xThomas
int16 yThomas
int16 xLightning
int16 yLightning
